website/docs/r/cis_waf_group.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "ibm"
+page_title: "IBM: cis_waf_group"
+sidebar_current: "docs-ibm-resource-cis-waf-group"
+description: |-
+  Provides an IBM Cloud Internet Services WAF Rule Group resource.
+---
+
+# ibm_cis_waf_group
+
+Provides a IBM CIS WAF Rule Group resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to change WAF Groups mode of a domain of a CIS instance
+
+## Example Usage
+
+```hcl
+resource "ibm_cis_waf_group" "test" {
+  cis_id     = data.ibm_cis.cis.id
+  domain_id  = data.ibm_cis_domain.cis_domain.domain_id
+  package_id = "c504870194831cd12c3fc0284f294abb"
```

Comment:
ids are hardcoded instead can we get from the resource?

Reply:
datasource is not planned for getting the package_id.  so hardcoded it